test/callbacks.js

```diff
@@ -68,6 +68,23 @@ QUnit.done( function() {
 			"testDone2",
 			"moduleDone1",
 			"moduleDone2",
+			"moduleStart1",
```

Comment:
I was kind of thrown by `moduleStart1` not actually relating to "module 1", but rather is the first callback invoked for module start. Any chance some of the callback text could be changed (maybe in a separate PR) to make it easier to follow when reviewing a file diff that doesn't show some of the context?